app/models/framework_response/multiple_item_object.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+class FrameworkResponse
+  class MultipleItemObject
+    include ActiveModel::Validations
+
+    attr_accessor :questions, :item, :responses, :person_escort_record
+
+    validates :item, presence: true, numericality: { only_integer: true }
+    validates :responses, presence: true
+    validate :validate_responses_type
+    validate :multiple_response_objects
+    validate :required_questions
+
+    def initialize(attributes: {}, questions: [], person_escort_record:)
+      attributes = attributes.presence || {}
+      @questions = questions
+      @person_escort_record = person_escort_record
+
+      attributes.deep_symbolize_keys! if attributes.respond_to?(:deep_symbolize_keys!)
+      @item = attributes[:item]
+      @responses = attributes[:responses] || []
+    end
+
+    def as_json(_options = {})
+      return {} unless item.present? && responses.any?
+
+      {
+        item: item,
+        responses: response_objects.map { |response| response_as_json(response) },
+      }
+    end
+
+  private
+
+    def multiple_response_objects
+      return unless responses_type_valid? && response_objects.any? { |object| object.invalid?(:update) }
+
+      response_objects.each_with_index do |object, index|
+        object.errors.each do |key, value|
+          errors.add("responses[#{index}].#{key}", value)
```

Comment:
This is based on https://blog.bigbinary.com/2016/07/07/errors-can-be-indexed-with-nested-attrbutes-in-rails-5.html